adoc/chapters/programming_interface.adoc

```diff
@@ -20865,7 +20865,7 @@ a@
 ----
 genfloat fmod (genfloat x, genfloat y)
 ----
-   a@ Modulus. Returns latexmath:[x \bmod y \cdot \mathrm{trunc}(x/y)].
+   a@ Modulus. Returns latexmath:[x \bmod y].
```

Comment:
These functions all come from OpenCL.  [OpenCL 3.0](https://www.khronos.org/registry/OpenCL/specs/3.0-unified/html/OpenCL_C.html#math-functions) defines it as:

> Modulus. Returns x - y * trunc(x/y).

where the `trunc` function is defined later (in both OpenCL and in SYCL).

This might be a better way to define the semantics, rather than defining it as  `x mod y`, because there is no formal definition in the spec for `mod`.

Reply:
Ah OK, yeah the OpenCL definition makes sense: looks like it was just a typo where '\bmod' repaced `-`. I can switch it to OpenCL def.

Reply:
Done

> Ah OK, yeah the OpenCL definition makes sense: looks like it was just a typo where '\bmod' repaced `-`. I can switch it to OpenCL def.

Done